packages/calcite-components/src/components/action/action.tsx

```diff
@@ -84,6 +88,17 @@ export class Action extends LitElement implements InteractiveComponent {
    */
   @property({ reflect: true }) dragHandle = false;
 
+  /**
+   * When `true` and the `type` is `"expand"`, the component is expanded.
+   */
+  @property({ reflect: true }) expanded = false;
+
+  /**
+   * When `true`, the component appears as if it is focused.
+   * @private
+   */
+  @property({ reflect: true }) focused = false;
```

Comment:
Nitpick - Shouldn't matter as its private - but is something like relatedActionMenuFocused more accurate

Reply:
Maybe `activeDescendant`? would be more accurate since that's what it is for?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-activedescendant